### ISSUE TITLE
Blob downloads failing on https://material-foundation.github.io/material-theme-builder/#/custom

### ISSUE THREAD

**Issue by ryanbr** [brave/brave-browser#23171]
## Description 
Downloading files from https://material-foundation.github.io/material-theme-builder/#/custom failing

## Steps to Reproduce

   1. Open `https://material-foundation.github.io/material-theme-builder/#/custom`
   2. Click on Export.
   3. Select an option, Android views

## Actual result:
![download-err](https://user-images.githubusercontent.com/1659004/171303467-44b9d054-b90f-4700-9536-dc92ba4e4f28.png)

## Expected result:
Should able to download

## Reproduces how often: 

Easily

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes 
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:
Works in Chrome and Firefox.

Download links are formatted `blob:null/d0accc4f-1ecb-41e5-a9da-9b7cf620739b`

**Comment by rkfg** [comment]
Same issue I guess? #23153

**Comment by kjozwiak** [comment]
Once the above is released via the `1.39.x` maintenance release, we'll revert https://github.com/brave/brave-variations/pull/270 & https://github.com/brave/brave-variations/pull/272 👍

**Comment by goodov** [comment]
> Once the above is released via the `1.39.x` maintenance release, we'll revert [brave/brave-variations#270](https://github.com/brave/brave-variations/pull/270) & [brave/brave-variations#272](https://github.com/brave/brave-variations/pull/272) 👍

We should not revert this. The proper way is to set `max_version`  into the study that doesn't include the fix. I'll do it, don't worry!

**Comment by LaurenWags** [comment]
Verified using

```
Brave | 1.39.118 Chromium: 102.0.5005.78 (Official Build) (x86_64)
-- | --
Revision | df6dbb5a9fd82af3f567198af2eb5fb4876ef99c-refs/branch-heads/5005_59@{#3}
OS | macOS Version 12.4 (Build 21F79)
```

Reproduced issue using `1.39.111` with `BravePartitionBlobStorage` enabled:

Example | Example
----- | -----
<img width="1189" alt="Screen Shot 2022-06-07 at 9 49 56 AM" src="https://user-images.githubusercontent.com/28145373/172397221-6bc74bed-c8a9-4c75-b4ac-b5ff54364fa6.png"> | <img width="1191" alt="Screen Shot 2022-06-07 at 9 49 45 AM" src="https://user-images.githubusercontent.com/28145373/172397256-8712ec80-6bd4-4a7b-8833-a37777bd5b89.png">

Verified with `1.39.118` and `BravePartitionBlobStorage` enabled that issue does not reproduce.
Also confirmed that https://dev-pages.brave.software/storage/partitioning.html is working as expected.

Example | Example | Example
----- | ----- | -----
<img width="1185" alt="enable1" src="https://user-images.githubusercontent.com/28145373/172397519-5ac69dce-5df0-4ce6-b2d7-0a624973907a.png"> | <img width="1188" alt="enable2" src="https://user-images.githubusercontent.com/28145373/172397535-a884c884-6751-4d0c-9998-36cffe6ad29d.png"> | <img width="1193" alt="enable3" src="https://user-images.githubusercontent.com/28145373/172397568-65fa0e95-17d7-4908-bcdf-0e121295e464.png">

Verified with `1.39.118` and `BravePartitionBlobStorage` disabled that issue does not reproduce.

Example | Example | Example
----- | ----- | -----
<img width="1190" alt="disable1" src="https://user-images.githubusercontent.com/28145373/172397805-be8577ea-505a-4682-9ce3-eff65ae394c8.png"> | <img width="1184" alt="disable2" src="https://user-images.githubusercontent.com/28145373/172397821-c9a09d7d-72f0-436c-b443-917217a8f995.png"> | <img width="1193" alt="disable3" src="https://user-images.githubusercontent.com/28145373/172397843-914e2081-eb37-48b9-a3af-3b61f696ae5a.png">

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.39.118 Chromium: 102.0.5005.78 (Official Build) (64-bit)
-- | --
Revision | df6dbb5a9fd82af3f567198af2eb5fb4876ef99c-refs/branch-heads/5005_59@{#3}
OS | Ubuntu 18.04 LTS

Reproduced the issue on 1.39.111

![image](https://user-images.githubusercontent.com/34715963/172438545-ee7590cb-6701-4882-af98-42563c19daf6.png)|![image](https://user-images.githubusercontent.com/34715963/172438676-370e261c-58d0-4b8b-9e6c-18f0ffd8349a.png)
--|--

Verified fixed in 1.39.118

![image](https://user-images.githubusercontent.com/34715963/172439193-64067d3c-2512-4580-89b6-66c447bbbff7.png)|![image](https://user-images.githubusercontent.com/34715963/172439493-e1035780-24fe-49b0-86c4-83c434a1f382.png)
--|--

Verified with 1.39.118 and BravePartitionBlobStorage disabled that issue does not reproduce

![image](https://user-images.githubusercontent.com/34715963/172439987-ca1f1c60-d404-4358-ac70-9a42ff62ce5f.png)|![image](https://user-images.githubusercontent.com/34715963/172440028-060e1960-b17d-40a2-8df4-977896337298.png)
--|--
